catalog-spec/catalog-spec.md

```diff
@@ -77,6 +77,30 @@ There are additional `rel` types in the [Using Relation Types](../best-practices
 they are more typically used in Collections, as catalogs tend to just be used to structure STAC organization, so tend to just use
 the ones above.
 
+## Media Types
+
+One of the best ways to help inform web clients about the content in a link is to use a common [Media 
+Type](https://en.wikipedia.org/wiki/Media_type) in the `type` field. In STAC the `type` field is used in both the 
+'[Link](#link-object)'' and '[Asset](../item-spec/item-spec.md#asset-object)' Objects. It is quite useful for STAC browsers to better determine
+what to render and display to users searching and browsing the catalog.  Media types are often referred to by the 
+now deprecated term "MIME types". 
+
+### STAC Media Types
+
+The following table lists the Media Types to use for STAC structures.
+
+| Media Type                     | Description                                                  |
+| ------------------------------ | ------------------------------------------------------------ |
+| `application/geo+json`	     | A STAC [Item](../item-spec/README.md)                        |
+| `application/json`             | A STAC [Catalog](#stac-catalog-specification)                |
+| `application/json`             | A STAC [Collection](../collection-spec/README.md)            |
+
+### Media Type for STAC Catalogs
```

Comment:
Maybe switch order of the chapters? After the table the media type definition for STAC Catalogs seems superfluous, but if you first define it and then give an overview table, it seems to have a better order.

Reply:
Sounds good.